packages/in_app_purchase/in_app_purchase_ios/lib/src/store_kit_wrappers/sk_payment_queue_wrapper.dart

```diff
@@ -450,6 +452,13 @@ class SKPaymentWrapper {
   /// testing.
   final bool simulatesAskToBuyInSandbox;
 
+  /// The details of optional discount that should be applied to the payment.
```

Comment:
Can we have a little more details here?
[Setting Up Promotional Offers] should cover most of the business logic on how to use this variable. We should also add more details on technical part.

For example, when will this value to be `null`

Reply:
Fixed.